geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-The Redis API for <%=vars.product_name%> allows <%=vars.product_name%> to function as a drop-in replacement for a
+The <%=vars.product_name%> API for Compatibility with Redis allows <%=vars.product_name%> to function as a drop-in replacement for a
 highly-available Redis data store, letting Redis applications take advantage of
-<%=vars.product_name%>’s scaling capabilities without changing their client code. Redis clients connect to a <%=vars.product_name%>
+<%=vars.product_name%>’s scaling capabilities without changing their client code. Compatible with Redis Clients connect to a <%=vars.product_name%>
```

Comment:
It might make sense to leave this as just `Redis clients` vs `Compatible with Redis clients` since we're talking about actual Redis clients (Jedis, Lettuce, etc.) ... then maybe we could specify that the server is a `Geode APIs compatible with Redis server`?